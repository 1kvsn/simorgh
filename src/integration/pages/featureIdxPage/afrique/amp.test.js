/**
 * @service afrique
 * @pathname /afrique/48465371
 */

import runAmpTests from '../ampTests';

describe('AMP afrique/48465371 Feature Index page', () => {
  runAmpTests(service);
});
