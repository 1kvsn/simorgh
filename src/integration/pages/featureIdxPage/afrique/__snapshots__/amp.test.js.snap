// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP afrique/48465371 Feature Index page A11y I can see the skip to content link 1`] = `"Aller au contenu"`;

exports[`AMP afrique/48465371 Feature Index page AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP afrique/48465371 Feature Index page AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
    "gbOrUnknown": Array [
      "gb",
      "gg",
      "im",
      "je",
      "uk",
      "unknown",
    ],
  },
}
`;

exports[`AMP afrique/48465371 Feature Index page Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=3&p=afrique.feature_index.48465371.page&r=%24%7BscreenWidth%7Dx%24%7BscreenHeight%7Dx%24%7BscreenColorDepth%7D&re=%24%7BavailableScreenWidth%7Dx%24%7BavailableScreenHeight%7D&hl=%24%7Btimestamp%7D&lng=%24%7BbrowserLanguage%7D&x1=[urn%3Abbc%3Acps%3A447a95b6-1c9f-e544-bf60-e23452e7fa71]&x2=[amp]&x3=[news-afrique]&x4=[fr]&x5=[%24%7BsourceUrl%7D]&x6=[%24%7BdocumentReferrer%7D]&x8=[simorgh]&x9=[Tout%2Bsavoir%2Bsur%2Bla%2BCAN%2B2019%2B-%2BBBC%2BNews%2BAfrique]&x11=[2019-05-30T14%3A23%3A38.000Z]&x12=[2019-07-19T12%3A46%3A18.000Z]&ref=%24%7BdocumentReferrer%7D",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP afrique/48465371 Feature Index page Analytics chartbeat 1`] = `
Object {
  "vars": Object {
    "contentType": null,
    "domain": "test.bbc.co.uk",
    "sections": "Afrique, Afrique - null",
    "title": null,
    "uid": 50924,
    "virtualReferrer": "\${documentReferrer}",
  },
}
`;

exports[`AMP afrique/48465371 Feature Index page Analytics comscore 1`] = `
Object {
  "extraUrlParams": Object {
    "comscorekw": "amp",
  },
  "vars": Object {
    "c2": "17986528",
  },
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/afrique"`;

exports[`AMP afrique/48465371 Feature Index page Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. La BBC n'est pas responsable du contenu des sites externes. Découvrez notre approche en matière de liens externes."`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 1`] = `
Object {
  "text": "BBC News, Afrique",
  "url": "/afrique",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 2`] = `
Object {
  "text": "Pourquoi vous pouvez faire confiance à BBC News",
  "url": "https://www.bbc.com/afrique/institutionelles-49283281",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 3`] = `
Object {
  "text": "Conditions d'utilisation",
  "url": "https://www.bbc.com/afrique/institutionelles-36826639",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 4`] = `
Object {
  "text": "A propos de la BBC",
  "url": "https://www.bbc.com/afrique/institutionelles-36826641",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 5`] = `
Object {
  "text": "Politique de confidentialité",
  "url": "https://www.bbc.com/afrique/institutionelles-36826642",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 6`] = `
Object {
  "text": "Cookies",
  "url": "https://www.bbc.com/usingthebbc/cookies/",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 7`] = `
Object {
  "text": "Contactez la BBC",
  "url": "https://www.bbc.com/afrique/institutionelles-36826644",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 8`] = `
Object {
  "text": "AdChoices / Do Not Sell My Info",
  "url": "https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/",
}
`;

exports[`AMP afrique/48465371 Feature Index page Footer Link should match text and url 9`] = `
Object {
  "text": "Découvrez notre approche en matière de liens externes.",
  "url": "https://www.bbc.co.uk/help/web/links/",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header I can see the branding 1`] = `"BBC News, Afrique"`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 1`] = `
Object {
  "text": "Accueil",
  "url": "/afrique",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 2`] = `
Object {
  "text": "Afrique",
  "url": "/afrique/region",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 3`] = `
Object {
  "text": "Monde",
  "url": "/afrique/monde",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 4`] = `
Object {
  "text": "Sports",
  "url": "/afrique/sports",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 5`] = `
Object {
  "text": "Economie",
  "url": "/afrique/topics/ca170ae3-99c1-48db-9b67-2866f85e7342",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 6`] = `
Object {
  "text": "Culture",
  "url": "/afrique/topics/6a73afa3-ea6b-45c1-80bb-49060b99f864",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 7`] = `
Object {
  "text": "Au féminin",
  "url": "/afrique/38058576",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 8`] = `
Object {
  "text": "Technologie",
  "url": "/afrique/topics/31684f19-84d6-41f6-b033-7ae08098572a",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 9`] = `
Object {
  "text": "Santé",
  "url": "/afrique/topics/c4794229-7f87-43ce-ac0a-6cfcd6d3cef2",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 10`] = `
Object {
  "text": "Vidéos",
  "url": "/afrique/media/video",
}
`;

exports[`AMP afrique/48465371 Feature Index page Header Navigation link should match text and url 11`] = `
Object {
  "text": "Nos émissions",
  "url": "/afrique/nos_emissions/radio",
}
`;

exports[`AMP afrique/48465371 Feature Index page Main heading should match text 1`] = `"Tout savoir sur la CAN 2019"`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 1`] = `
Object {
  "sizes": null,
  "url": "http://localhost:7080/afrique/images/icons/icon-192x192.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 2`] = `
Object {
  "sizes": "72x72",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-72x72.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 3`] = `
Object {
  "sizes": "96x96",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-96x96.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 4`] = `
Object {
  "sizes": "128x128",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-128x128.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 5`] = `
Object {
  "sizes": "144x144",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-144x144.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 6`] = `
Object {
  "sizes": "152x152",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-152x152.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 7`] = `
Object {
  "sizes": "180x180",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-180x180.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 8`] = `
Object {
  "sizes": "192x192",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-192x192.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 9`] = `
Object {
  "sizes": "384x384",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-384x384.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Apple Touch Icon should match attributes 10`] = `
Object {
  "sizes": "512x512",
  "url": "https://news.files.bbci.co.uk/include/articles/public/afrique/images/icons/icon-512x512.png",
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO Canonical link 1`] = `"http://localhost:7080/afrique/48465371"`;

exports[`AMP afrique/48465371 Feature Index page SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP afrique/48465371 Feature Index page SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP afrique/48465371 Feature Index page SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP afrique/48465371 Feature Index page SEO Heading level 1 1`] = `"Tout savoir sur la CAN 2019"`;

exports[`AMP afrique/48465371 Feature Index page SEO Lang attribute 1`] = `"fr"`;

exports[`AMP afrique/48465371 Feature Index page SEO Linked data should match text 1`] = `
Object {
  "@context": "http://schema.org",
  "@graph": Array [
    Object {
      "@type": "WebPage",
      "image": Object {
        "@type": "ImageObject",
        "height": 576,
        "url": "https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png",
        "width": 1024,
      },
      "mainEntityOfPage": Object {
        "@id": "http://localhost:7080/afrique/48465371",
        "@type": "WebPage",
        "name": "Tout savoir sur la CAN 2019",
      },
      "publisher": Object {
        "@type": "NewsMediaOrganization",
        "logo": Object {
          "@type": "ImageObject",
          "height": 576,
          "url": "https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png",
          "width": 1024,
        },
        "name": "BBC News Afrique",
        "publishingPrinciples": "https://www.bbc.com/afrique/institutionelles-49283281",
      },
      "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png",
      "url": "http://localhost:7080/afrique/48465371",
    },
  ],
}
`;

exports[`AMP afrique/48465371 Feature Index page SEO OG description 1`] = `"La Coupe d'Afrique des nations, compétition de football la plus importante dans le continent, se joue en Egypte du 21 juin au 19 juillet 2019."`;

exports[`AMP afrique/48465371 Feature Index page SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png"`;

exports[`AMP afrique/48465371 Feature Index page SEO OG image alt 1`] = `"BBC News Afrique"`;

exports[`AMP afrique/48465371 Feature Index page SEO OG locale 1`] = `"fr"`;

exports[`AMP afrique/48465371 Feature Index page SEO OG site name 1`] = `"BBC News Afrique"`;

exports[`AMP afrique/48465371 Feature Index page SEO OG title 1`] = `"Tout savoir sur la CAN 2019 - BBC News Afrique"`;

exports[`AMP afrique/48465371 Feature Index page SEO OG type 1`] = `"website"`;

exports[`AMP afrique/48465371 Feature Index page SEO OG url 1`] = `"http://localhost:7080/afrique/48465371"`;

exports[`AMP afrique/48465371 Feature Index page SEO Page title 1`] = `"Tout savoir sur la CAN 2019 - BBC News Afrique"`;

exports[`AMP afrique/48465371 Feature Index page SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP afrique/48465371 Feature Index page SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP afrique/48465371 Feature Index page SEO Twitter creator 1`] = `"@bbcafrique"`;

exports[`AMP afrique/48465371 Feature Index page SEO Twitter description 1`] = `"La Coupe d'Afrique des nations, compétition de football la plus importante dans le continent, se joue en Egypte du 21 juin au 19 juillet 2019."`;

exports[`AMP afrique/48465371 Feature Index page SEO Twitter image alt 1`] = `"BBC News Afrique"`;

exports[`AMP afrique/48465371 Feature Index page SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/afrique.png"`;

exports[`AMP afrique/48465371 Feature Index page SEO Twitter site 1`] = `"@bbcafrique"`;

exports[`AMP afrique/48465371 Feature Index page SEO Twitter title 1`] = `"Tout savoir sur la CAN 2019 - BBC News Afrique"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 1`] = `"CAN 2019 : le Sénégal qualifié pour les huitièmes de finale"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 2`] = `"CAN 2019 : le Maroc termine premier de son groupe"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 3`] = `"Vidéo, \\"Qu’il pleuve ou qu’il neige, comptez sur nous\\", 0,46"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 4`] = `"Madagascar s'offre un 8e historique"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 5`] = `"CAN 2019 : pour les huitièmes, le Cameroun va encore attendre"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 6`] = `"CAN 2019 : la réintégration de l'égyptien Warda ne fait pas l'unanimité"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 7`] = `"CAN 2019 : un bon début pour le Cameroun"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 8`] = `"CAN 2019 : tout savoir des Aigles du Mali"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 9`] = `"L'Ouganda à l'assaut des \\"fimbu\\" de la RDC"`;

exports[`AMP afrique/48465371 Feature Index page Sections should match headline text 10`] = `"Can 2019 : l'Algérie, une équipe à surveiller de près"`;

exports[`AMP afrique/48465371 Feature Index page Story Promo Headline should match text 1`] = `"CAN 2019 : le Sénégal qualifié pour les huitièmes de finale"`;

exports[`AMP afrique/48465371 Feature Index page Story Promo Image should match image 1`] = `
<amp-img
  alt="CAN 2019 : le Sénégal en huitième de finale"
  attribution="Getty Images"
  height="549"
  layout="responsive"
  src="https://ichef.bbci.co.uk/news/660/cpsprodpb/4826/production/_107707481_equipesenegal.jpg"
  srcset="https://ichef.bbci.co.uk/news/70/cpsprodpb/4826/production/_107707481_equipesenegal.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsprodpb/4826/production/_107707481_equipesenegal.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsprodpb/4826/production/_107707481_equipesenegal.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsprodpb/4826/production/_107707481_equipesenegal.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsprodpb/4826/production/_107707481_equipesenegal.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsprodpb/4826/production/_107707481_equipesenegal.jpg 320w, https://ichef.bbci.co.uk/news/660/cpsprodpb/4826/production/_107707481_equipesenegal.jpg 660w"
  width="976"
/>
`;
